pkg/flatrpc/conn.go

```diff
@@ -191,7 +191,7 @@ func (c *Conn) recv(size int) error {
 	return nil
 }
 
-func verify(raw any, rawSize int) error {
+func Verify(raw any, rawSize int) error {
```

Comment:
Nit:
I wonder if it had been cleaner to let `RecvRaw[T]` accept bytes and move most of `RecvRaw` to `Recv`. Then we could have reused more code - e.g. there would be no `parseExecResult()`.

Reply:
Done. PTAL.